src/apps/prestashop/mod.rs

```diff
@@ -0,0 +1,57 @@
+use crate::Runner;
+use crate::{Action, State};
+use async_trait::async_trait;
+use thirtyfour::prelude::*;
+
+pub struct T();
+
+#[async_trait]
+impl Runner for T {
+    async fn exec(&self, st: &State) -> WebDriverResult<()> {
+        match &st.act {
+            Action::Test => {
+                let _u = st.url.clone();
```

Comment:
intended?

Reply:
This is what you get when you try out the new ``cargo clippy --fix``. In my head I was thinking "I know there's nothing dangerous it could do, warning for unnecessary var/clone is a great use case for this". I didn't think it would generate provably dead code, how silly.